ksql-rest-model/src/main/java/io/confluent/ksql/rest/entity/FunctionType.java

```diff
@@ -15,4 +15,8 @@
 
 package io.confluent.ksql.rest.entity;
 
-public enum FunctionType { scalar, aggregate }
+public enum FunctionType {
+  scalar,
```

Comment:
again not your code, but can this follow the convention that enums should be all caps? I'm surprised this isn't in checkstyle so I guess you could argue that it's not a project rule but I don't think I've ever seen lowercase enums before!

Reply:
changed